cohort_4/playbook/SCHEDULE.md

```diff
@@ -64,12 +64,12 @@ will be followed by project-time that day.
 | 8:30 AM  | Standups                                                     |
 | 9:20 AM  | [Hacker arrival](ACTIVITIES.md#hacker-arrival)               |
 | 9:30 AM  | Galvanize tour                                               |
-| 9:40 AM  | Energizer                                                    |
+| 9:40 AM  | [Energizer 1](ACTIVITIES.md#where-the-west-wind-blows) & [Energizer 2](#i-love-my-neighbor-who-___)                                 |
```

Comment:
@zachlatta 

- [ ] I can't figure out how to make the link work for [Energizer 2](#i-love-my-neighbor-who-___) and